vcd/resource_vcd_vapp_vm.go

```diff
@@ -397,6 +397,7 @@ func resourceVcdVAppVmUpdate(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("error getting VM status: %#v", err)
 	}
 
+	// all functionality using retryCall as multi requests to same vAPP isn't handled and errors occur
```

Comment:
```suggestion
	// When there is more then one VM in a vApp Terraform will try to parallelise their creation.
	// However, vApp throws errors when simultaneous requests are executed. To avoid them, below block is using retryCall in multiple places as a workaround, so that the VMs are created regardless of parallelisation.
```
(not fmt'ed though)

Reply:
Fixed